sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/autorest.md

```diff
@@ -0,0 +1,24 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+```yaml
+directive:
+- from: swagger-document
+  where: $.definitions.*
+  transform: >
+    $["x-csharp-usage"] = "model,input,output";
+    $["x-csharp-formats"] = "json";
+
+require: https://github.com/Azure/azure-rest-api-specs/blob/55b3e2d075398ec62f9322829494ff6a4323e299/specification/videoanalyzer/data-plane/readme.md
+azure-arm: false
+payload-flattening-threshold: 2
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: Microsoft.Azure.Media.VideoAnalyzer.Edge
```

Comment:
1. This namespace isn't correct, as track 2 uses Azure.*.
2. This isn't necessary since the generator uses the package directory name anyway. At the very least, I would author this to match to avoid confusion, but just removing it would be better.